stored_requests/backends/db_provider/mysql_dbprovider.go

```diff
@@ -75,7 +84,65 @@ func (provider *MySqlDbProvider) ConnString() string {
 		buffer.WriteString(provider.cfg.Database)
 	}
 
-	return buffer.String()
+	// TSL connection
+	if provider.cfg.ServerCa != "" && provider.cfg.ClientCert != "" && provider.cfg.ClientKey != "" {
```

Comment:
If we created a TLS struct as I described in an earlier comment, and all three of these are needed for TLS, then we could put a receiver method on that TLS struct that checks if TLS is explicitly enabled.